src/Containers/App/index.jsx

```diff
@@ -283,7 +283,7 @@ class App extends Component {
       if(result) {
         result.index = target;
         result.ref = React.createRef();
-        individual[target] = result;
+        individual[i] = result;
```

Comment:
Careful: `i` is the position in the target array, not in the individual array. Here we should find the index of the individual ESC matching the actual target.

Reply:
I see. Here I should use `escs.individual.findIndex((esc) => esc.index === target)`. I will update soon after I hava this tested.